### DISCUSSION TITLE
这是什么问题啊QAQ

### DISCUSSION THREAD

**Discussion by 2187Kai** [LmeSzinc/AzurLaneAutoScript#2688] [Q&A]
![屏幕截图 2023-06-02 013612](https://github.com/LmeSzinc/AzurLaneAutoScript/assets/135260696/7aa405ee-8f3e-4ba1-8822-bb872d1ae0ab)

**Comment by LmeSzinc** [comment]
https://github.com/LmeSzinc/AzurLaneAutoScript/issues/2687#issuecomment-1572526474

**Reply by 2187Kai** [reply]
嗷，解决了，谢谢！
